cmd/operator/main.go

```diff
@@ -83,10 +81,15 @@ var (
 )
 
 func init() {
+	defaultAddr := ":8080"
+	if v := os.Getenv("PORT"); v != "" {
+		defaultAddr = fmt.Sprintf(":%s", v)
+	}
+
 	flag.StringVar(&flLoggingLevel, "verbosity", "info", "the logging level (e.g. debug)")
 	flag.BoolVar(&flCLI, "cli", false, "run as CLI application to manage rollout in intervals")
 	flag.IntVar(&flCLILoopIntervalSec, "cli-run-interval", 60, "the time between each rollout process (in seconds)")
-	flag.StringVar(&flHTTPAddr, "http-addr", "", "listen on http portrun on request (e.g. :8080)")
+	flag.StringVar(&flHTTPAddr, "http-addr", defaultAddr, "port where to listen to http requests (e.g. 8080)")
 	flag.StringVar(&flProject, "project", "", "project in which the service is deployed")
```

Comment:
on user's computer we can define project as `gcloud config get-value core/project -q` + trim output. 

on Cloud Run (by looking at $K_SERVICE) https://github.com/GoogleCloudPlatform/cloud-run-hello/blob/9b0e68f416d5f3f23398bad81d57f1e268a99ad8/hello.go#L37-L52

if you detect something successfully: log it user-visible.